### PR TITLE
Stop Making Me Hurt You

### PR DESCRIPTION
**Pull request by ParadoxalObserver**
-Reduce Prospector and Salvager income to none. Foreman remains unchanged. 
-Reduces departmental funds from 10600 to 6500.
-Now with 3/3 Headadmin approval, k5
-#possumsvision

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/game/jobs/department.dm [hunk 1]
```diff
@@ -125,7 +125,7 @@
 	name = "Prospectors"
 	id = DEPARTMENT_PROSPECTOR
 	//Nep in 5 hours with full team is 10600
-	account_initial_balance = 10600 //Starts with an even wage to cover payment-expenses. Their job is to make money and, thus, should not start with more money than needed.
+	account_initial_balance = 5000 //Starts with an even wage to cover payment-expenses. Their job is to make money and, thus, should not start with more money than needed.
```
Comment (Trilbyspaceclone) [inline code/game/jobs/department.dm:128]:
```suggestion
	account_initial_balance = 6500 //Starts with an even wage to cover payment-expenses. Their job is to make money and, thus, should not start with more money than needed.
```

File: code/game/jobs/department.dm [hunk 2]
```diff
@@ -125,7 +125,7 @@
 	name = "Prospectors"
 	id = DEPARTMENT_PROSPECTOR
 	//Nep in 5 hours with full team is 10600
```
Comment (Trilbyspaceclone) [inline code/game/jobs/department.dm:127]: Please fix this commit and redo the math

**Comment by ParadoxalObserver** [comment]
image

**Comment by Rat-Time** [comment]
This again?  Assuming it's from the guy that cries about prospector players/gameplay and said he'd hang around sabotaging the department with nerfs.

**Comment by ParadoxalObserver** [comment]
> This again? Assuming it's from the guy that cries about prospector players/gameplay and said he'd hang around sabotaging the department with nerfs.

The prior one was closed due to a mistake by k5. He read the meme, salt PR and believed that one was mine. After talking to him and the headmins and getting things cleared up, this got the okay. Stop being so salty.

File: code/game/jobs/department.dm (hunk 2 above)
Comment (Trilbyspaceclone) [reply code/game/jobs/department.dm:127]: This commit should be fixed

**Comment by ApplePlastic** [comment]
image_2023-03-16_212333094

**Comment by Trilbyspaceclone** [comment]
All0 headmins agreed and this has been sitting for a bit I have no reason to not approve it

**Pull request merged**